nhentai/doujinshi.py

```diff
@@ -72,8 +72,14 @@ def __repr__(self):
     def show(self):
         logger.info(f'Print doujinshi information of {self.id}\n{tabulate(self.table)}')
 
-    def download(self, regenerate_cbz=False, file_type=''):
+    def download(self, skip_exists=True):
         logger.info(f'Starting to download doujinshi: {self.name}')
+
+        base_path = os.path.join(self.downloader.path, self.filename)
+        if (os.path.exists(base_path + '.pdf') or os.path.exists(base_path + '.cbz')) and skip_exists:
+            logger.info(f'Skip download doujinshi because a PDF/CBZ file exists of doujinshi {self.name}')
```

Comment:
In fact, this code will never be reached, will be removed in the future

Reply:
I was able to reach this code with the following commands:
```
nhentai --id 530833 --download --format '%t [%i]' --threads=4 --pdf --meta --output="test-out"
nhentai --id 530833 --download --format '%t [%i]' --threads=4 --cbz --meta --output="test-out"
```